common/djangoapps/student/views.py

```diff
@@ -1558,6 +1571,7 @@ def create_account_with_params(request, params):
     # don't send email if we are doing load testing or random user generation for some reason
     # or external auth with bypass activated
     send_email = (
+        not settings.FEATURES.get('SKIP_EMAIL_VALIDATION', None) and
```

Comment:
It is not. If unset, the get(,None) returns None, and it defaults to the old behavior (require activation).